package.json

```diff
@@ -8,6 +8,7 @@
   },
   "scripts": {
     "node-test": "mocha test/**/*-test.js",
+    "browser-test": "COVERAGE=true ember test",
```

Comment:
Yeah, I see it now [here](https://github.com/kategengler/ember-cli-code-coverage/blob/master/test/integration/coverage-test.js#L23-L28)